docs/csharp/language-reference/operators/null-coalescing-operator.md

```diff
@@ -22,7 +22,7 @@ Available in C# 8.0 and later, the null-coalescing assignment operator `??=` ass
 
 The left-hand operand of the `??=` operator must be a variable, a [property](../../programming-guide/classes-and-structs/properties.md), or an [indexer](../../programming-guide/indexers/index.md) element.
 
-In C# 7.3 and earlier, the type of the left-hand operand of the `??` operator must be either a reference type or a [nullable value type](../../programming-guide/nullable-types/index.md). Beginning with C# 8.0, that requirement is replaced with the following: the type of the left-hand operand of the `??` and `??=` operators cannot be a non-nullable value type. In particular, beginning with C# 8.0, you can use the null-coalescing operators with unconstrained type parameters:
+In C# 7.3 and earlier, the type of the left-hand operand of the `??` operator must be either a [reference type](../keywords/reference-types.md) or a [nullable value type](../builtin-types/nullable-value-types.md). Beginning with C# 8.0, that requirement is replaced with the following: the type of the left-hand operand of the `??` and `??=` operators cannot be a non-nullable value type. In particular, beginning with C# 8.0, you can use the null-coalescing operators with unconstrained type parameters:
```

Comment:
nit: I'd add the quotes to make it clear where the text from the spec ends:

```suggestion
In C# 7.3 and earlier, the type of the left-hand operand of the `??` operator must be either a [reference type](../keywords/reference-types.md) or a [nullable value type](../builtin-types/nullable-value-types.md). Beginning with C# 8.0, that requirement is replaced with the following: "the type of the left-hand operand of the `??` and `??=` operators cannot be a non-nullable value type." In particular, beginning with C# 8.0, you can use the null-coalescing operators with unconstrained type parameters:
```

Reply:
@BillWagner would it be fine to quote not direct spec text? The [updated spec text (check the end of the linked section)](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/null-coalescing-assignment#detailed-design) reads:

> If A exists and is a non-nullable value type, a compile-time error occurs.

where A - is the type of the left-hand operand.